http/responseemitter.go

```diff
@@ -8,14 +8,14 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/ipfs/go-ipfs-cmds"
+	cmds "github.com/ipfs/go-ipfs-cmds"
 )
 
 var (
-	HeadRequest = fmt.Errorf("HEAD request")
```

Comment:
I think this is the only exported thing that is changing. I don't think it is used anywhere in our codebases. Seemed ok to remove.